bears/r/FormatRBear.py

```diff
@@ -15,8 +16,38 @@ class FormatRBear:
     LANGUAGES = "R"
 
     @staticmethod
-    def create_arguments(filename, file, config_file):
-
-        rcode = ("formatR::tidy_source(source =\""+filename+"\"")
+    def create_arguments(filename, file, config_file,
+                         r_comment: bool = True,
+                         r_blank: bool = True,
+                         r_brace: bool = False,
+                         r_arrow: bool = False,
+                         tab_width: int=SpacingHelper.DEFAULT_TAB_WIDTH,
+                         max_expr_length: int=20):
+        """
+        :param r_comment:       Use to determine whether comments are
+                                kept or not
+        :param r_blank:         Use to determine whether blank lines
+                                are kept or not
+        :param r_brace:         Whether to put the left brace
+        :param r_arrow:         Whether to replace the assign operator = with <-
+        :param tab_width:       Number of space for indentation
+        :param max_expr_length: Maximum number of characters for an expression
+                                Note (different from the maximum number of
+                                characters for a line).
+        """
+        rcode = ("formatR::tidy_source("
+                 "source =\"{filename}\","
+                 "comment ={r_comment},"
+                 "blank ={r_blank},"
+                 "arrow ={r_arrow},"
+                 "width.cutoff ={max_expr_length},"
+                 "brace.newline ={r_brace},"
+                 "indent ={tab_width}"
+                 ")".format(filename=filename, tab_width=tab_width,
+                            r_comment="TRUE" if r_comment else "FALSE",
+                            r_blank="TRUE" if r_blank else "FALSE",
+                            r_arrow="TRUE" if r_arrow else "FALSE",
+                            r_brace="TRUE" if r_brace else "FALSE",
```

Comment:
maybe make a little function out of that? `_map_to_r_bool` or something like that? but it's up to you, I think this version is also okay and clear